### PR TITLE
Update yaml example

### PR DESCRIPTION
**Pull request by crtahlin**
- moved yaml example to separate file (now included in the main file)
- renamed flags for two options, see below

changes:
```
# enable debug HTTP API (true / false)
enable-debug-api : false
# disable P2P QUIC protocol (enable / disable)
p2p-disable-quic: disable
# disable P2P WebSocket protocol (enable / disable)
p2p-disable-ws: disable
# enable tracing (true / false)
tracing: false
```

### CODE REVIEW

File: content/bee-docs/start/config-yaml-template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# NAT exposed address
+nat-addr: ""
+# ID of the Swarm network (default 1)
+network-id: 1
+# P2P listen address (default ":7070")
+p2p-addr: :7070
+# disable P2P QUIC protocol (enable / disable)
+p2p-disable-quic: disable
+# disable P2P WebSocket protocol (enable / disable)
+p2p-disable-ws: disable
```
Comment (janos) [inline content/bee-docs/start/config-yaml-template.md:27]:
```suggestion
# enable P2P QUIC protocol (true / false)
p2p-enable-quic: false
# enable P2P WebSocket protocol (true / false)
p2p-enable-ws: false
```

**Comment by crtahlin** [comment]
Wait with this PR, some things will not go into master ! (probably)

**Comment by Eknir** [comment]
I think this one can be closed

**Comment by significance** [comment]
superseeded

**Pull request closed without merging**